### ISSUE TITLE
connection problem

### ISSUE THREAD

**Issue by Byblack36** [qwqdanchun/DcRat#47]
Hello, dc-rat connection arrives in 30 minutes or 1 hour, sometimes it takes longer

**Comment by qwqdanchun** [comment]
have you test on your own pc



---Original---
From: ***@***.***&gt;
Date: Fri, Aug 6, 2021 21:52 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: [qwqdanchun/DcRat] connection problem (#47)




 
Hello, dc-rat connection arrives in 30 minutes or 1 hour, sometimes it takes longer

**Comment by Byblack36** [comment]
yes , there is a file currently working on another pc, but it comes late

**Comment by qwqdanchun** [comment]
this is so strange



---Original---
From: ***@***.***&gt;
Date: Fri, Aug 6, 2021 22:05 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [qwqdanchun/DcRat] connection problem (#47)




 
yes , there is a file currently working on another pc, but it comes late

**Comment by Byblack36** [comment]
when the pc restarts the connection comes immediately

**Comment by Byblack36** [comment]
but the client must be open

**Comment by qwqdanchun** [comment]
can't know why. that's nor normal

**Comment by Byblack36** [comment]
what do you suggest i do

**Comment by Byblack36** [comment]
By the way, the version I'm using is 1.0.7

**Comment by qwqdanchun** [comment]
try on another pc and test if this happened again



---Original---
From: ***@***.***&gt;
Date: Sat, Aug 7, 2021 01:42 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [qwqdanchun/DcRat] connection problem (#47)




 
what do you suggest i do

**Comment by Byblack36** [comment]
ok i will try

**Comment by Byblack36** [comment]
![DC](https://user-images.githubusercontent.com/44087037/128560074-2d3a0ea2-c66a-42e1-a2a7-af7aec7128a0.png)

If I don't use the install option, won't it be permanent?

**Comment by qwqdanchun** [comment]
yes



---Original---
From: ***@***.***&gt;
Date: Sat, Aug 7, 2021 03:10 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [qwqdanchun/DcRat] connection problem (#47)




 

 
If I don't use the install option, won't it be permanent?

**Comment by Byblack36** [comment]
how can i build shell code
What should I choose the project as and build it in visual studio?
current encoder

When I transfer the decoder, I do not get a proper directory

Can you advise on encryption?

**Comment by qwqdanchun** [comment]
open the project in vs2019 and choose release and build



---Original---
From: ***@***.***&gt;
Date: Mon, Aug 9, 2021 00:59 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [qwqdanchun/DcRat] connection problem (#47)




 
how can i build shell code
 What should I choose the project as and build it in visual studio?
 current encoder
 
When I transfer the decoder, I do not get a proper directory
 
Can you advise on encryption?

**Comment by Byblack36** [comment]
In the installation tab, I say normal installation, it does not give any response.
i say schtask install
the process says it is being used by another process

**Comment by Byblack36** [comment]
keyboard control works but mouse control does not work mouse does not move
